build.gradle

```diff
@@ -81,6 +82,20 @@ def findPropertyOrEnv(String key) {
   [project.properties[key], System.getenv(key)].find { it != null }
 }
 
+artifactory {
+    contextUrl = 'https://oss.jfrog.org'
+    publish {
+        repository {
+            repoKey = 'oss-snapshot-local'
+            username = findPropertyOrEnv("BINTRAY_USER") ?: "no.bintray.user"
+            password = findPropertyOrEnv("BINTRAY_API_KEY") ?: "no.bintray.api.key"
+        }
+        defaults {
+            publications 'HoodPublication'
+        }
+    }
+}
+
 bintray {
```

Comment:
If bintray publication has been deleted a lot of unused code is here

Reply:
Right, good point !! :+1: